json/json.go

```diff
@@ -19,3 +24,20 @@ func Stringify(v interface{}, opts ...interface{}) string {
 	}
 	return string(buf)
 }
+
+// ReadFile Tries to open a file from the file system and read it into an object
+func ReadFile(f string, out interface{}) error {
+	if !fileutil.FileExists(f) {
+		return errors.New("File not found " + f)
+	}
```

Comment:
no need to change but you can use `fmt.Errorf` which allows you to do something like `fmt.Errorf("File not found %s", f)` instead and returns an error